src/App.js

```diff
@@ -1,13 +1,46 @@
 import React from 'react';
-
 import './App.scss';
+import { GoodList } from './components/GoodList/GoodList';
+import { getAll, get5FirstHandler, getRedHandler } from './api/goods';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+  selectGoods = (responseGoods) => {
+    responseGoods()
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  };
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  render() {
+    return (
+      <div>
+        <h1>Dynamic list of Goods</h1>
+        <GoodList goods={this.state.goods} />
+        <button
+          onClick={() => this.selectGoods(getAll)}
+          type="button"
+        >
+          getAll
```

Comment:
```suggestion
          Get all
```
It's just text on the button. You can name it whatever you like. No need for camelCase